docs/tutorials/typescript/digital-ocean-deployment/index.mdx

```diff
@@ -0,0 +1,283 @@
+---
+id: index
+title: Deploy a TypeScript application to production on a Digital Ocean Droplet
+description: Learn how to deploy your TypeScript-based Temporal Application to a DigitalOcean Droplet using Ubuntu 20.04 or 22.04
+sidebar_label: Deploy a TypeScript application to production on a Digital Ocean Droplet
+tags:
+  - typescript
+  - deployment
+  - digital-ocean
+  - temporal-cloud
+keywords:
+  - TypeScript Temporal Application deployment
+  - DigitalOcean Droplet setup
+  - Ubuntu 20.04 server setup
+  - Ubuntu 22.04 server setup
+  - Install Node.js on Ubuntu
+  - Install TypeScript on Ubuntu
+  - SSH setup DigitalOcean
+  - Custom domain configuration
+  - Nginx reverse proxy setup
+  - Let's Encrypt SSL certificate
+  - PM2 process manager
+  - Temporal Cloud integration
+  - Deploy TypeScript application DigitalOcean
+  - Temporal Cloud certificates
+---
+
+When you are ready to deploy your TypeScript based Temporal Application to production, you need a server to host it.
+Digital Ocean is provides flexible Cloud servers called Droplets that can be used to host your application.
+
+In this tutorial we will walk through the steps to deploy your Temporal Application to a Digital Ocean Droplet while using Temporal Cloud as your Temporal Application's supervisor.
```

Comment:
"supervisor" isn't language I've seen often here, and if this is a Cloud tutorial, I think that should be in the title, not DigitalOcean.

Moreover -- "Ensure you can connect to the Temporal Service" should probably be a prerequisite tutorial that we have on hand and we can link to before we publish this tutorial. Perhaps one option for self-hosted and one option for Cloud? That way this is scoped better.

Reply:
I disagree on the first point and to be honest I don't understand the second point.

Reply:
OK, understood about the SEO argument for the first point.

For the second point: I think there's a lot of general "make sure you can SSH / make sure the Cloud handshake works in the most basic ways" stuff in this tutorial. If you'd rather keep it in, I understand why, but then I think we should reframe the introduction here slightly so that the audience knows that this is all about being introduced to Cloud.